Changelog.md

```diff
@@ -15,7 +15,7 @@ This changelog keeps track of work items that have been completed and are ready
 
 ### Breaking Changes
```

Comment:
I don't think this is breaking though

Reply:
it isn't indeed, not after the revert of registry changes